External/Plugins/CodeRefactor/PluginMain.cs

```diff
@@ -418,8 +418,10 @@ private void RenameClicked(Object sender, EventArgs e)
         {
             try
             {
-                Rename command = new Rename(true);
-                command.Execute();
+                //InlineRename.CancelExisting();
+                //new Rename(useInline);
+                ////or
+                InlineRename.Start();
```

Comment:
Uncommenting the top 2 lines and deleting the last line could be a better option...
`useInline` can just be a constant value or it can be from settings.